docs/examples/general/index.rst

```diff
@@ -1,12 +1,16 @@
-General
-=======
-
 .. toctree::
    :maxdepth: 2
 
    data_loading/index
    expressions/index
    multigpu.ipynb
+
+General Purpose Operations
+==========================
```

Comment:
Meh, it added a header in the middle of a page: http://dali-ci-01:7070/docs/2067617/html/examples/general/index.html